apps/web/src/components/GetStarted/BuildWithUsFooter.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import { titleClasses, linkTextClasses } from '../../../app/(base-org)/getstarted/page';
+import { ButtonWithLinkAndEventLogging } from '../Button/ButtonWithLinkAndEventLogging';
+import { ButtonVariants } from '../Button/Button';
+import buildWithUs from './images/build-with-us.png';
+
+export default async function BuildWithUsFooter() {
+  const imageBackgroundStyles = {
+    backgroundImage: `url('${buildWithUs.src}')`,
+  };
+
+  const containerClasses = classNames(
```

Comment:
Doesn't seem like any reason for these to live inside of the render function since they seem static

Reply:
moved all classes outside of the render functions